frontend/.husky/commit-msg

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+cd frontend && yarn commitlint --edit $1
```

Comment:
Add newline

Reply:
didn't get this one

Reply:
Add new line at the end of file _(with Axolo)_

Reply:
got it _(with Axolo)_